modules/filter/context.cpp

```diff
@@ -63,6 +63,7 @@ namespace {
     
   PosibErr<bool> ContextFilter::setup(Config * config){
     name_ = "context-filter";
+    order_num_ = 0.15;
```

Comment:
Changing the order number is likely to break something.  If it needs to be changed than it should be an option to the context filer, something like "order-num \<float\>" where float is a number between 0 and 1.

Reply:
At present, `order_num` is not defined at all for the context filter. What does that mean? Is the filter applied first, last or somewhere else? (If I knew the answer 12 years ago when I wrote this patch, I've forgotten it in the meantime.)

Reply:
It defaults to 0.50:  https://github.com/GNUAspell/aspell/blob/875068a6bcff884d62b29c3215c2b3304ab4b774/common/indiv_filter.hpp#L92

Reply:
@kevina I understand your concern that this may break something. For the new TeX mode, however, it is important that the context filter comes first. So it would have to be added as an option. Unfortunately, my familiarity with the aspell code is (and never was) sufficient to add such an option. Would you be willing to help out?

Reply:
Yes.  It shouldn't be too hard to modify the context filter to add the option.